src/CMakeLists.txt

```diff
@@ -56,18 +54,27 @@ sentry_target_sources_cwd(sentry
 # generic platform / path / symbolizer
 if(WIN32)
 	sentry_target_sources_cwd(sentry
+		sentry_random.c
+		sentry_random.h
 		sentry_windows_dbghelp.c
 		sentry_windows_dbghelp.h
 		path/sentry_path_windows.c
 		symbolizer/sentry_symbolizer_windows.c
 	)
+elseif(NX)
+	sentry_target_sources_cwd(sentry
+		sentry_unix_spinlock.h
+		path/sentry_path_unix.c
```

Comment:
Okay, so just to be clear, you will also add your random implementation in a build script downstream, not here. You could refer here to a file that does not exist upstream.

Reply:
Yes, we're including sentry-native in the downstream cmake & adding sources to the original target using `target_sources(sentry ...), currently there's backend, random, path, modulefinder and os. 
I don't really want to enumerate them downstream as there will be changes in their relative locations, names, langauge (c/cpp), etc. so I wouldn't want to have to touch upstream just to do that.